synth.py

```diff
@@ -47,21 +48,25 @@
 
```

Comment:
Do we still need to exclude the `src/index.ts`? And seems we reformat the `synth.py`, is this from some tool?

Reply:
Good catch. Removed.